src/microbe_stage/editor/CellStatsIndicator.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Globalization;
+using Godot;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Shows cell stats (e.g. Storage: 2.1, Hp: 50, etc) for the organism statistics display.
+///   Also functions as a comparison for old value with a new one, indicated with an up/down icon.
+/// </summary>
+public class CellStatsIndicator : HBoxContainer
+{
+    private Label? descriptionLabel;
+    private Label? valueLabel;
+    private TextureRect? changeIndicator;
+
+    private Texture increaseIcon = null!;
+    private Texture decreaseIcon = null!;
+    private Texture questionIcon = null!;
+
+    private string description = "unset";
+    private string format = string.Empty;
```

Comment:
I think using `string? ` very good idea here. Null is after all a special value so it's very easy to make this clear that null means "don't use format".